README.md

```diff
@@ -20,143 +20,20 @@ _Thanks to the [Better Jinja](https://github.com/samuelcolvin/jinjahtml-vscode)
 - Block comment toggling: Press `CMD + /` to create HubL comments
 - Block level indentation: Text inside of `{% block %}` will be indented automatically
 
-### __HubL Snippets__
-All HubL supported tags, filters, expression tests and functions have auto-complete snippets. Expression tests are accessed by typing the test name alone, filters are accessed with `|` and functions/tags are accessed with `~`. All snippets include descriptions and parameter details. You up/down arrow to navigate the IntelliSense and hit enter to execute a snippet. Snippet completed HubL statements will auto-highlight available parameters, which can be tabbed through (`${parameter}`).
+### HubL Snippets
+All HubL supported tags, filters, expression tests and functions have auto-complete snippets. Expression tests are accessed by typing the test name alone, filters are accessed with `|` and functions/tags are accessed with `~`. All snippets include descriptions and parameter details. You up/down arrow to navigate the IntelliSense and hit enter to execute a snippet. Snippet completed HubL statements will auto-highlight available parameters, which can be tabbed through (`${parameter}`). Additionally there is support for other commonly needed syntax, such as module fields.
 
-![Parameters](https://cdn2.hubspot.net/hubfs/2359872/IMPORTANT/DONOTDELETE/hubl-language-extension/params.png)
-[__HubL Tags__](https://designers.hubspot.com/docs/hubl/hubl-supported-tags) produce entire HubL tag statements with available parameters. Ex `~he` > Enter produces:
-```
-{% header "${my_header}"
-   header_tag="${header_tag}",
-   value="${value}"
-%}
-```
-[__HubL Filters__](https://designers.hubspot.com/docs/hubl/hubl-supported-filters) produce entire HubL filter statements with available parameters. Ex `|se` > Enter produces:
-```
-|selectattr("${attr}", ${exp_test})
-```
-[__HubL Functions__](https://designers.hubspot.com/en/docs/hubl/hubl-supported-functions) produce entire HubL function statements with available parameters, without wrapping curly braces. The intention of this is so you can use HubL functions within other HubL statements easily (like setting variables, for loops, etc.) Ex `~hub` > Enter produces:
-```
-hubdb_table_rows(${table_id}, ${query})
-```
-[__HubL Expression Tests__](https://designers.hubspot.com/docs/hubl/operators-and-expression-tests#expression-tests) produce expression test names. Ex `di` > Enter produces:
-```
-divisibleby
-```
+## Getting Started
```

Comment:
This does not seem to be quite how it works for me. I need to search for something more like "Files: Associations" to get the setting to show up. I also see "Edit in settings.json" rather than "add item"

<img width="898" alt="Screen Shot 2020-09-18 at 9 42 48 AM" src="https://user-images.githubusercontent.com/35824747/93604593-9267b400-f993-11ea-968a-c4eeb099a2e9.png">

Reply:
Which version of VS Code are you running?

Reply:
I was on 1.30.1 , after upgrading this is working now

Reply:
I figured that was the case. Looks like the UI was updated in July 2019 so I'm not too worried about documenting the old way unless anyone disagrees